tools/src/main/scala/scala/scalanative/build/Discover.scala

```diff
@@ -19,6 +19,9 @@ object Discover {
   def optimize(): Boolean =
     getenv("SCALANATIVE_OPTIMIZE").forall(_.toBoolean)
 
+  def multithreadingSupport(): Boolean =
+    getenv("SCALANATIVE_MULTITHREADING_ENABLED").exists(_.toBoolean)
```

Comment:
What do you think about `SCALANATIVE_MULTITHREADING` which is consistent with `SCALANATIVE_OPTIMIZE` ?